src/cmd/property.go

```diff
@@ -0,0 +1,116 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/opslevel/opslevel-go/v2023"
+
+	"github.com/opslevel/cli/common"
+
+	"github.com/spf13/cobra"
+)
+
+var createPropertyDefinitonCmd = &cobra.Command{
+	Use:   "property-definition",
+	Short: "Create a property-definition",
+	Long:  `Create a property-definition`,
+	Example: fmt.Sprintf(`
+cat << EOF | opslevel create property-definition -f -
+%s
+EOF`, getExamplePropertyDefinitionYaml()),
+	Run: func(cmd *cobra.Command, args []string) {
+		input, err := readResourceInput[opslevel.PropertyDefinitionInput]()
+		cobra.CheckErr(err)
+		newPropertyDefinition, err := getClientGQL().CreatePropertyDefinition(*input)
+		cobra.CheckErr(err)
+
+		fmt.Println(newPropertyDefinition.Id)
+	},
+}
+
+var getPropertyDefinition = &cobra.Command{
+	Use:        "property-definition",
+	Short:      "Get details about a property definition",
+	Long:       `Get details about a property definition`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID"},
+	Run: func(cmd *cobra.Command, args []string) {
+		identifier := args[0]
+		result, err := getClientGQL().GetPropertyDefinition(identifier)
+		cobra.CheckErr(err)
+
+		if isYamlOutput() {
+			common.YamlPrint(result)
+		} else {
+			common.PrettyPrint(result)
+		}
+	},
+}
+
+var genPropertyDefinition = &cobra.Command{
+	Use:   "property-definition",
+	Short: "Generate example yaml of a Property Definition",
+	Long:  `Generate example yaml of a Property Definition`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println(getExamplePropertyDefinitionYaml())
+	},
+}
+
+func getExamplePropertyDefinitionYaml() string {
+	schema := `{"$schema":"https://json-schema.org/draft/2020-12/schema", "type": "boolean"}`
```

Comment:
we should use a more featureful example - can you reachout to foundations and ask what examples/templates they are putting in the UI and put one of the more complex ones here?

Reply:
Definitely! The more feature rich example will live in opslevel-go